fire/api/model/tidsserier.py

```diff
@@ -424,6 +440,475 @@ def residualkovarians_zz(self) -> List[float]:
         """
         return self._obs_liste(ResidualKovarians, "zz")
 
+    def binning(
```

Comment:
Som vi talte om , er det nødvendigt for en statisk metode at have Python-decorator'en `staticmethod` for at sikre, at den også virker som statisk metode på instanser af klassen, da en instans af klassen vil give instansen selv som det første argument, når metoden bliver kaldt på instansen. (Det er nemlig ligegyldigt, hvad man kalder første argument på ikke-statiske metoder på en klasse. `self` (og `cls`, hvis man bruger decorator'en `classmethod`) er bare konventioner, der imidlertid gør kode nemmere at læse.

```suggestion
    @staticmethod
    def binning(
```